### ISSUE TITLE
Github 2 Jake

### ISSUE THREAD

**Issue by JakeCode121** [JakeCode121/Jakes-Stuff#1]

**Comment by haggis78** [comment]
Hi Jake. This is to confirm that I'm on your repo now.

**Comment by JakeCode121** [comment]
Thanks, Dr. Campbell.  I feel like a robot vacuum, who is just bouncing around trying to find a open space.

**Comment by haggis78** [comment]
Yeah, I get that. Next up in XML world is writing schemas, which is a kind of puzzle. Each new task brings its own challenges, and its own metaphors.
